src/connection.ts

```diff
@@ -518,7 +518,7 @@ class Connection extends EventEmitter {
         enableImplicitTransactions: false,
         enableNumericRoundabort: false,
         enableQuotedIdentifier: true,
-        encrypt: false,
+        encrypt: true,
```

Comment:
Oh this is purely for testing purposes so we can see the packets being sent and being recieved. we can leave it false for now, and set it back to `true` when we can submit this branch to the actual tedious master